### PR TITLE
fix(all-the-things): For eternity

### PR DESCRIPTION
**Pull request by oleorhagen**
Ticket: HAL-9000
Changelog: I wish you all the best of luck, gentlemen. Fare well!

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -1,244 +1,10 @@
...
-    - apt-get update
-    - apt install mender-artifact
-    - export CC=$(which clang)
-    - export CXX=$(which clang++)
   script:
-    - cmake
-      -D CMAKE_BUILD_TYPE=${BUILD_TYPE}
-      .
-    - make --jobs=$(nproc --all) --keep-going check
+    - true
```
Comment (lluiscampos) [inline .gitlab-ci.yml:8]: The CI run is failing with `jobs:test:unit:script config should be a string or a nested array of strings up to 10 levels deep`

`true` is a boolean value in YAML and it is not a valid entry for `script`. You can fix it with:
```suggestion
    - "true"
```

**Review by lluiscampos — CHANGES_REQUESTED** [review]
This needs still some work, see comment below.

**Review by kacf — COMMENTED** [review]
I really like this approach! Clean and tidy! 😂

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -1,244 +1,10 @@
...
-  extends: .test:static-template
-  script:
-    - cmake -D BUILD_TESTS=OFF .
-    - make --jobs=$(nproc --all) --keep-going
-  tags:
-    - mender-qa-worker-generic
 
 test:unit:
   stage: test
   image: debian:11
```
Comment (alfrunes) [inline .gitlab-ci.yml:6]: Nitpick: Use busybox for security hardening.
```suggestion
  image: busybox:stable-musl
```

File: .gitlab-ci.yml (hunk 1 above)
Comment (alfrunes) [reply .gitlab-ci.yml:8]: Could also consider removing the `true` statement for increased performance.
```yaml
  script: []
```

**Review by alfrunes — COMMENTED** [review]
I admire the effort to remove technical debt. Well done @oleorhagen! :clap:
I just have a couple small nitpicks...

**Comment by lluiscampos** [comment]
Closing this one for now. Feel free to submit again in a later point - we always welcome external contributions :heart:

**Pull request closed without merging**